rclcpp_lifecycle/include/rclcpp_lifecycle/state.hpp

```diff
@@ -21,8 +21,8 @@
 
 #include "rcutils/allocator.h"
 
-// forward declare rcl_state_t
-typedef struct rcl_lifecycle_state_t rcl_lifecycle_state_t;
+// forward declare rcl_lifecycle_state_t
+typedef struct rcl_lifecycle_state_s rcl_lifecycle_state_t;
```

Comment:
Can we just remove this completely and `#include <rcl_lifecycle/data_types.h>`?

Reply:
Sure. See https://github.com/ros2/rclcpp/pull/1788/commits/d1fb7db56e8e6cc43c9f0a9cd675997bee8aa690.